examples/fluent-bit/health-check/README.md

```diff
@@ -42,6 +42,12 @@ If you do not use the built-in configuration for this health check, you will nee
     HC_Period 5
 ```
 
+An alternative would be to use an include statement to import the built-in config from your config file:
```

Comment:
Since this file also includes a [SERVICE] section do we need to note that here as done with internal metrics?

Reply:
good point... should've thought of that when I wrote it for the other one...